xml/deployment_preparation.xml

```diff
@@ -300,37 +338,18 @@ suse_caasp:
     For example, to create the ISO image for an admin node on a computer
     running &opensuse; 42:
    </para>
-   <screen>&prompt.user;sudo genisoimage -output <replaceable>cc-admin.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-admin</replaceable></screen>
+   <screen>&prompt.user;<command>sudo genisoimage -output <replaceable>cc-admin.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-admin</replaceable></command></screen>
```

Comment:
IMO better would be to use .iso extension "cc-admin.iso" + the same for cc-worker image... Line 347
... we have an chapter "Preparing an ISO image" but here you create an image with "img" extension.